balena.yml

```diff
@@ -15,5 +26,9 @@ applicationEnvironmentVariables:
     - WINDOW_POSITION: 0,0
     - LOCAL_HTTP_DELAY: 0
     - FBCP_DISPLAY: ""
+  defaultDeviceType: "raspberry-pi-64"
+  supportedDeviceTypes:
+    - "raspberrypi4-64"
```

Comment:
could you please add  `fincm3`, `raspberrypi3` , `raspberrypi3-64`, `intel-nuc` , `genericx86-64-ext` ?